scripts/modernExtendRefactor.ts

```diff
@@ -60,10 +60,10 @@ project.getSourceFiles().forEach((sourceFile) => {
                     const evalOpts = Object.entries(eval(`(${opts})`));
                     for (const [key, value] of evalOpts) {
                         if (key === 'colorTempRange') {
-                            // @ts-expect-error
+                            // @ts-expect-error ignore
                             newOpts.colorTemp = {...newOpts.colorTemp, range: value};
```

Comment:
`newOpts.colorTemp as object` to get rid of the `@ts-expect-error`

Reply:
Done!